exonum-java-binding/core/rust/src/proxy/executor.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use jni::objects::JObject;
+use jni::{JNIEnv, JavaVM};
+
+use std::sync::Arc;
+
+use JniResult;
+
+/// The capacity of local frames, allocated for attached threads
```

Comment:
… by default. Same as the default value Hotspot uses when calling native Java methods.